cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-hcp.adoc

```diff
@@ -0,0 +1,139 @@
++
+[source,terminal]
+----
+curl https://raw.githubusercontent.com/openshift-cs/rosaworkshop/master/rosa-workshop/rosa/resources/setup-vpc.sh | bash
+----
++
+For more about VPC requirements, see the xref:../../../rosa_planning/rosa-sts-aws-prereqs.adoc#rosa-vpc_rosa-sts-aws-prereqs[VPC documentation].
+
+. The above script outputs two commands. Set the commands as environment variables to make running the `create cluster` command easier. Copy them from the output and run them as shown:
++
+[source,terminal]
+----
+export PUBLIC_SUBNET_ID=<public subnet id here>
+export PRIVATE_SUBNET_ID=<private subnet id here>
+----
+
+. Confirm that the environment variables are set by running the following command:
++
+[source,terminal]
+----
+echo "Public Subnet: $PUBLIC_SUBNET_ID"; echo "Private Subnet: $PRIVATE_SUBNET_ID"
+----
++
+.Example output
++
+[source,terminal]
+----
+Public Subnet: subnet-0faeeeb0000000000
+Private Subnet: subnet-011fe340000000000
+----
+
+=== Creating your OIDC configuration
+
+In this workshop, we will use the automatic mode when creating the OIDC configuration. We will also store the OIDC ID as an environment variable for later use. The command uses the ROSA CLI to create your cluster's unique OIDC configuration. 
+
+* To create the OIDC configuration for this workshop, run the following command:
++
+[source,terminal]
+----
+export OIDC_ID=$(rosa create oidc-config --mode auto --managed --yes -o json | jq -r '.id')
+----
+
+=== Creating additional environment variables
+
+* Run the following command to set up some environment variables so that it is easier to run the command to create the ROSA with HCP cluster:
++
+[source,terminal]
+----
+export CLUSTER_NAME=<enter cluster name>
+export REGION=<region VPC was created in>
+----
+
+== Creating the cluster
+If this is the _first time_ you are deploying ROSA in this account and you have _not_ yet created the account roles, create the account-wide roles and policies, including the Operator policies. Since ROSA uses AWS Security Token Service (STS), this step creates the AWS IAM roles and policies that are needed for ROSA to interact with your account.
+
+. Run the following command to create the account-wide roles:
++
+[source,terminal]
+----
+rosa create account-roles --mode auto --yes
+----
+
+. Run the following command to create the cluster:
++
+[source,terminal]
+----
+rosa create cluster --cluster-name $CLUSTER_NAME \
+    --subnet-ids ${PUBLIC_SUBNET_ID},${PRIVATE_SUBNET_ID} \
+    --hosted-cp \
+    --region $REGION \
+    --oidc-config-id $OIDC_ID \
+    --sts --mode auto --yes
+----
+
+The control plane and API are ready after about 10 minutes and the worker nodes and cluster are ready and completely useable after about 15 to 20 minutes. The cluster will have a control plane across three AWS availability zones in your selected region and create two worker nodes in your AWS account.  
```

Comment:
That conflicts with Provisioning time mentioned in:
 https://68291--docspreview.netlify.app/openshift-rosa/latest/rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly#rosa-hcp-classic-comparison_rosa-hcp-sts-creating-a-cluster-quickly